uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/manager-leave/components/manager-leave-calendar.js

```diff
@@ -110,6 +111,8 @@ define([
      * @param  {Array} leaveRequests - leave requests array from API
      */
     vm._indexLeaveRequests = function (leaveRequests) {
+      vm.leaveRequests = {};
```

Comment:
1. At the beginning you are initializing `vm.leaveRequests` as an array (`vm.leaveRequests = []`), but now you reset it as an object? Something's not right
2. Why the re-initialization in the first place? It's not clear why it is necessary